media/css/mozorg/contribute.scss

```diff
@@ -101,6 +101,7 @@ $image-path: '/media/protocol/img';
         .contribute-card {
             display: flex;
             flex: 1 0 45%;
+            overflow-wrap: break-word;
```

Comment:
also considered `hyphens: auto` but it would be a progressive enhancement as [support is spotty](https://developer.mozilla.org/en-US/docs/Web/CSS/hyphens#browser_compatibility) and I'm not sure it's a better experience as more words end up broken onto two lines
<img width="308" alt="hyphens" src="https://user-images.githubusercontent.com/19650432/200554359-57ee5a5d-d9aa-407a-87b0-640b02c33822.png">